sketchcrapp.sh

```diff
@@ -146,31 +161,32 @@ genSelfSignCert() {
 importSelfSignCert() {
   userKeyChain="$(security default-keychain -d user | sed -e 's/^[ ]*//g' -e 's/\"//g')"
   if ! [ -f "$userKeyChain" ]; then
-    echo "User default Keychain does not exist: $userKeyChain"
+    echo "[-] User default Keychain does not exist: $userKeyChain"
     exit 1
   fi
+  echo "[+] Importing private key and self-signed certificate" 
   security import pkcs.p12 -k "$userKeyChain" -f pkcs12 -P 1234
 }
 
 # Equivalent to code-signature application in Sketch. Sign Sketch with generated
 # certificate.
 signApplication() {
   appPath="$1"
-  echo "Signing the patched *.app bundle. This may require sudo."
-  echo "Enter your Login password if a dialogue pops-up, and remember to choose Always allow."
+  echo "[+] Signing the patched *.app bundle. This may require sudo."
+  echo "[+] Enter your Login password if a dialogue pops-up, and remember to choose Always allow."
```

Comment:
`[+] Enter your Login password if a dialogue pops-up, and remember to choose Always allow.` => `[+] If asked, enter your login password. Choose "Always Allow" to not be asked again.`